### PR TITLE
feat(core): add track events for page option

### PR DESCRIPTION
**Pull request by JimmFly**

### CODE REVIEW

**Comment by JimmFly** [comment]
* **#7668** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/7668?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#7667** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/7667?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#7664** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/7664?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#7661** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/7661?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#7735** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/7735?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#7695** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/7695?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `canary`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @JimmFly and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/frontend/core/src/modules/explorer/views/nodes/doc/index.tsx [hunk 1]
```diff
@@ -41,9 +41,11 @@ export const ExplorerDocNode = ({
   canDrop,
   operations: additionalOperations,
   dropEffect,
+  explorerTreesType,
```
Comment (EYHN) [inline packages/frontend/core/src/modules/explorer/views/nodes/doc/index.tsx]: I think we can just delete this and let each node be based on its own type

File: packages/frontend/core/src/components/page-list/operation-cell.tsx [hunk 2]
```diff
@@ -113,13 +118,26 @@ export const PageOperationCell = ({
...
     duplicate(page.id, false);
     track.allDocs.list.docMenu.createDoc({
       control: 'duplicate',
     });
   }, [duplicate, page.id]);
 
+  const handleRemoveFromAllowList = useCallback(() => {
+    if (onRemoveFromAllowList) {
+      onRemoveFromAllowList();
+      track.collection.docList.docMenu.removeSpecialFilter();
```
Comment (forehalo) [inline packages/frontend/core/src/components/page-list/operation-cell.tsx]: deleteOrganizeItem({ type: 'doc' })

Comment (JimmFly) [reply packages/frontend/core/src/components/page-list/operation-cell.tsx]: But it is not `organizeItem`, this function is to remove doc from collection.

File: packages/frontend/core/src/mixpanel/events.ts [hunk 3]
```diff
@@ -57,14 +53,16 @@ type CollectionEvents =
   | 'createCollection'
   | 'deleteCollection'
   | 'renameCollection'
-  | 'addDocToCollection';
+  | 'addDocToCollection'
+  | 'removeDocFromCollection';
```
Comment (forehalo) [inline packages/frontend/core/src/mixpanel/events.ts]: use `removeOrganizeItem({ type: 'doc' })`

**Review by forehalo — APPROVED** [review]

**Pull request merged**